src/github/issue-comment.ts

```diff
@@ -18,7 +18,7 @@ export const issueCommentWebhookHandler = async (
 	} = context.payload;
 	let linkifiedBody;
 
-	const gitHubInstallationClient = await createInstallationClient(githubInstallationId, jiraClient.baseURL, context.log);
+	const gitHubInstallationClient = await createInstallationClient(githubInstallationId, context.log, jiraClient.baseURL);
```

Comment:
We're now using the `gitHubInstallationId` to check against entries in the Subscriptions table. The `jiraHost` param in the 3 client factory functions is only used to set the FF. Seeing this will later be removed, I moved all `jiraHost` args so that they are the last to be passed.